ViewModel/ImpactAreaScenario/Results/PerformanceVMBase.cs

```diff
@@ -1,27 +1,34 @@
 ﻿using metrics;
 using System.Collections.Generic;
 using HEC.FDA.ViewModel.ImpactAreaScenario.Results.RowItems;
+using HEC.Plotting.SciChart2D.DataModel;
 
 namespace HEC.FDA.ViewModel.ImpactAreaScenario.Results
 {
     public abstract class PerformanceVMBase : BaseViewModel
     {
         private List<IPerformanceRowItem> _rows;
 
-        public Dictionary<Threshold, List<IPerformanceRowItem>> MetricsToRows { get; set; }
-
+        public Dictionary<Threshold, List<IPerformanceRowItem>> MetricsToRows { get; set; } = new Dictionary<Threshold, List<IPerformanceRowItem>>();
```

Comment:
Should the setter be public or should you just remove it?

Reply:
I think you are right, i can get rid of the setter. It is gone.